V2.0/mailin.py

```diff
@@ -20,15 +18,16 @@ def do_request(self,resource,method,indata):
 
     if self.timeout is not None and (self.timeout <=0 or self.timeout > 60):
       raise Exception('value not allowed for timeout')
-    
-    #request timeout
-    socket.setdefaulttimeout(self.timeout)
 
-    h = httplib2.Http(".cache", disable_ssl_certificate_validation=True)
-    content_type = "application/json"
-    r,c = h.request(url,method,body=indata,headers={'api-key':self.api_key, 'content-type':content_type})
-    charenc = chardet.detect(c)
-    return json.loads(c.decode(charenc['encoding']))  
+    content_type = 'application/json'
+    headers = {
+        'api-key': self.api_key,
```

Comment:
This part of the code uses 4-space indentation while the rest uses 2-space indentation.